From: Mark Nelson <mnelson@redhat.com>
Date: Thu, 21 Sep 2017 17:21:16 -0500
Subject: common/options.cc: Set Filestore rocksdb compaction readahead option.

fixes: http://tracker.ceph.com/issues/21505

Signed-off-by: Mark Nelson <mnelson@redhat.com>
(cherry picked from commit 66567e573836c570040c8d8148c34b0cf9dc9ce2)

Conflicts:
	src/common/options.cc (moved to config_opts.h)

Resolves: rhbz#1462011
---
 src/common/config_opts.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common/config_opts.h b/src/common/config_opts.h
index 6c34ce6..9a67811 100644
--- a/src/common/config_opts.h
+++ b/src/common/config_opts.h
@@ -864,7 +864,7 @@ OPTION(rocksdb_collect_extended_stats, OPT_BOOL, false) //For rocksdb, this beha
 OPTION(rocksdb_collect_memory_stats, OPT_BOOL, false) //For rocksdb, this behavior will be an overhead of 5%~10%, collected only rocksdb_perf is enabled.
 
 // rocksdb options that will be used for omap(if omap_backend is rocksdb)
-OPTION(filestore_rocksdb_options, OPT_STR, "")
+OPTION(filestore_rocksdb_options, OPT_STR, "compaction_readahead_size=2097152")
 // rocksdb options that will be used in monstore
 OPTION(mon_rocksdb_options, OPT_STR, "cache_size=536870912,write_buffer_size=33554432,block_size=65536,compression=kNoCompression")
 
